contrib/inventory/packet_net.py

```diff
@@ -0,0 +1,491 @@
+#!/usr/bin/env python
+
+'''
+Packet.net external inventory script
+=================================
+
+Generates inventory that Ansible can understand by making API request to
+Packet.net using the Packet library.
+
+NOTE: This script assumes Ansible is being executed where the environment
+variable needed for Packet API KEY have already been set:
+    export PACKET_NET_API_KEY='PK123'
```

Comment:
This is not a comment per se, but a suggestion:
Would you consider changing the name of the env var from  PACKET_NET_API_KEY to PACKET_API_TOKEN? In Packet terms it is a "token":
https://www.packet.net/developers/api/authentication/

..and, we got 2 modules for Packet merged to devel, where I refer to the API token as PACKET_API_TOKEN. It would be nice to the users to have the API token in the same envvar for inventory and modules. The merged PR with the modules is here:
https://github.com/ansible/ansible/pull/19005